PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install CrowdStrike Falcon Agent.md

```diff
@@ -0,0 +1,60 @@
+#### Name
+
+Windows - Install CrowdStrike Falcon Agent | v1.0 JCCG
+
+#### commandType
+
+windows
+
+#### Command
+
+```
+$CID="ENTER CID HERE"
+$installerURL="ENTER URL TO FALCON AGENT INSTALLER HERE"
+
+##### Do Not Edit Below This Line #####
+
+$installerTempLocation="C:\Windows\Temp\CSFalconAgentInstaller.exe"
```

Comment:
Do we want to delete this file once done?

Reply:
I don't think we ever really came to a conclusion on that decision. My thought is that, (while I know this is a script), installers generally don't delete themselves when finished.  Also, this is being saved in the Temp folder, which means it can safely be deleted if needed.

I really don't have a strong opinion either way, but my preference would be to just let it be. Adding extra steps to delete it adds more complexity that I don't think is necessary.

Reply:
Whatever we leave in %Temp% will eventually be cleaned up by the system. Accidentally deleting something unintentional would be worse than letting the system clean up this on next reboot.